charts/spire/charts/spire-server/templates/statefulset.yaml

```diff
@@ -189,6 +191,14 @@ spec:
             - name: spire-data
               mountPath: /run/spire/data
               readOnly: false
+            {{- if or (eq (.Values.tornjak.config.connectionType | toString) "tls") (eq (.Values.tornjak.config.connectionType | toString) "mtls") }}
+            - name: server-cert
+              mountPath: /opt/spire/server
```

Comment:
should we store all certs in the same folder?

That is how it normally works on webservers.

- ca.pem
- server.pem

Also wonder if there should be a user cert here.

Normally in a PKI setup your server and client cert would be signed with the same CA.

The client cert is normally just on the client if I'm not mistaking.

Reply:
The client cert is just on the clients, yes. The user ca is coming from a separate object, so better to have a separate mount IMO.